http/src/main/java/io/micronaut/http/ssl/ClientSslConfiguration.java

```diff
@@ -36,6 +36,10 @@ public class ClientSslConfiguration extends AbstractClientSslConfiguration {
      */
     public static final String PREFIX = "micronaut.http.client.ssl";
 
+    private boolean disableHostnameVerification;
+
```

Comment:
Remove these empty lines.

Also checkstyle will complain about this, you can run it locally with `./gradlew check`